lib/debootstrap.sh

```diff
@@ -668,13 +676,11 @@ PREPARE_IMAGE_SIZE
 		fi
 
 		check_loop_device "$rootdevice"
-		display_alert "Creating rootfs" "$ROOTFS_TYPE on $rootdevice"
-		mkfs.${mkfs[$ROOTFS_TYPE]} ${mkopts[$ROOTFS_TYPE]} ${mkopts_label[$ROOTFS_TYPE]:+${mkopts_label[$ROOTFS_TYPE]}"$ROOT_FS_LABEL"} $rootdevice >> "${DEST}"/${LOG_SUBPATH}/install.log 2>&1
+		display_alert "Creating root fs" "${ROOTFS_TYPE} on ${rootdevice}"
+		mkfs.${mkfs[$ROOTFS_TYPE]} ${mkopts[$ROOTFS_TYPE]=:''} ${mkopts_label[$ROOTFS_TYPE]:+${mkopts_label[$ROOTFS_TYPE]}${ROOT_FS_LABEL}} ${rootdevice} >> ${DEST}/${LOG_SUBPATH}/install.log 2>&1
```

Comment:
What is `=:`?

Reply:
a typo, should be := and is there to suppress any errors thrown when the variable is unset.
And without the '' as those are given literally when unset.